src/components/newTodoForm/newTodoForm.tsx

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+/* eslint-disable react-hooks/rules-of-hooks */
+import React, { ChangeEvent, FormEvent, useState } from 'react';
+import { Todo, User } from '../../types';
+import classNames from 'classnames';
+// import { todo } from 'node:test';
```

Comment:
The import statement for `todo` from 'node:test' is commented out and seems unnecessary. Consider removing it if it's not needed.